Dockerfile.tmpl

```diff
@@ -591,7 +591,7 @@ ARG BUILD_DATE=unknown
 
 LABEL git-commit=$GIT_COMMIT
```

Comment:
@Philmod Does the fact that this may now contain brackets change how this is used at all?

Reply:
It contains brackets because the `{` was before the `$`. It shouldn't contain these brackets anymore

Reply:
👍  i just meant this other reference that also uses `$GIT_COMMIT` (`LABEL`, different than the one you changed). by it's name i imagine it's just cosmetic, but was curious.